LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
q: is this in sync w/ other OS projects that we have or do we have a mix of i.e. Apache vs. MIT?

Reply:
I spot-checked a bunch of criblio GitHub repos, and I see a mix of Apache and MIT.